operations/mimir-tests/test-read-write-deployment-mode-s3-generated.yaml

```diff
@@ -981,6 +981,7 @@ spec:
         - -server.grpc.keepalive.min-time-between-pings=10s
         - -server.grpc.keepalive.ping-without-stream-allowed=true
         - -server.http-listen-port=8080
+        - -store-gateway.sharding-ring.auto-forget-enabled=false
```

Comment:
This is interesting. We shouldn't change it here. I think you need to apply the change only when deployment mode is microservices.